### PR TITLE
Update org.apache.zookeeper:* to v3.9.0

### PR DESCRIPTION
**Pull request by solrbot**
This PR contains the following updates:

| Package | Type | Update | Change |
|---|---|---|---|
| [org.apache.zookeeper:zookeeper-jute](http://zookeeper.apache.org) ([source](https://gitbox.apache.org/repos/asf/zookeeper.git)) | dependencies | minor | `3.8.1` -> `3.9.0` |
| [org.apache.zookeeper:zookeeper](http://zookeeper.apache.org) ([source](https://gitbox.apache.org/repos/asf/zookeeper.git)) | dependencies | minor | `3.8.1` -> `3.9.0` |

---

### Configuration

📅 **Schedule**: Branch creation - "* * * * 0" (UTC), Automerge - At any time (no schedule defined).

🚦 **Automerge**: Disabled by config. Please merge this manually once you are satisfied.

♻ **Rebasing**: Whenever PR becomes conflicted, or you tick the rebase/retry checkbox.

🔕 **Ignore**: Close this PR and you won't be reminded about these updates again.

---

 - [ ] If you want to rebase/retry this PR, check this box

---

This PR has been generated by [Renovate Bot](https://togithub.com/solrbot/renovate-github-action)

### CODE REVIEW

**Comment by janhoy** [comment]
Fails with

- License file missing ('io.netty:netty-tcnative-boringssl-static:2.0.61.Final'), expected it at: /home/runner/work/solr/solr/solr/licenses/netty-tcnative-boringssl-static-LICENSE-[type].txt or /home/runner/work/solr/solr/solr/licenses/netty-tcnative-boringssl-LICENSE-[type].txt or /home/runner/work/solr/solr/solr/licenses/netty-tcnative-LICENSE-[type].txt or /home/runner/work/solr/solr/solr/licenses/netty-LICENSE-[type].txt, where [type] can be any of [ASL, BSD, BSD_LIKE, CDDL, CPL, EPL, MIT, MPL, PD, SUN, COMPOUND].

**Comment by janhoy** [comment]
I compared LICENSE and NOTICE files for all the different `netty` jars, and they are all identical. So I deleted the jar specific ones and replaced with common `netty-LICENSE-ASL.txt` and `netty-NOTICE.txt`.

File: versions.lock [hunk 1]
```diff
@@ -114,12 +114,14 @@ io.netty:netty-codec-http:4.1.96.Final (3 constraints: 582436ee)
 io.netty:netty-codec-http2:4.1.96.Final (1 constraints: ed0a94cb)
 io.netty:netty-codec-socks:4.1.96.Final (1 constraints: 160fb96c)
 io.netty:netty-common:4.1.96.Final (12 constraints: 3cb3817d)
-io.netty:netty-handler:4.1.96.Final (3 constraints: 732b553d)
+io.netty:netty-handler:4.1.96.Final (3 constraints: 722b4f3d)
 io.netty:netty-handler-proxy:4.1.96.Final (1 constraints: ed0a94cb)
 io.netty:netty-resolver:4.1.96.Final (2 constraints: 5c1a8332)
+io.netty:netty-tcnative-boringssl-static:2.0.61.Final (1 constraints: d10fc38e)
+io.netty:netty-tcnative-classes:2.0.61.Final (1 constraints: d113ea5d)
```
Comment (janhoy) [inline versions.lock:121]: `tcnative-boringssl` is pulled in due to https://issues.apache.org/jira/browse/ZOOKEEPER-4622 which adds more performant SSL support. Do you think we can safely exclude this dependency from Solr? I could not read from the JIRA whether this is opt-in or applies to the zk client, server or both.

Comment (risdenk) [reply versions.lock:121]: I don't think we need to exclude anything. I think that dependency is just a shim? Its a few KB if that - https://mvnrepository.com/artifact/io.netty/netty-tcnative-boringssl-static/2.0.61.Final

Based on reading ZOOKEEPER-4622 it looks like the native libraries have to be setup separately (os and classpath).

**Review by risdenk — APPROVED** [review]

File: versions.lock (hunk 1 above)
Comment (janhoy) [reply versions.lock:121]: Yea, no big deal with such light weight. Agree.

**Pull request merged**